src/axom/mint/examples/user_guide/mint_tutorial.cpp

```diff
@@ -494,399 +461,393 @@ void fem_tutorial()
+  mint::write_vtk(&mesh, "tutorial_triangle_mesh.vtk");
 }
 
 //------------------------------------------------------------------------------
-void construct_mixed_cell_type_unstructured( )
+void construct_mixed_cell_type_unstructured()
 {
-
-// sphinx_tutorial_construct_unstructured_mixed_start
+  // sphinx_tutorial_construct_unstructured_mixed_start
 
   constexpr int DIMENSION = 2;
 
   // Construct the mesh object
-  mint::UnstructuredMesh< mint::MIXED_SHAPE > mesh( DIMENSION );
+  mint::UnstructuredMesh<mint::MIXED_SHAPE> mesh(DIMENSION);
 
   // Append the mesh nodes
-  const axom::IndexType n0 = mesh.appendNode( 0.0, 0.0 );
-  const axom::IndexType n1 = mesh.appendNode( 2.0, 0.0 );
-  const axom::IndexType n2 = mesh.appendNode( 1.0, 1.0 );
-  const axom::IndexType n3 = mesh.appendNode( 3.5, 1.0 );
-  const axom::IndexType n4 = mesh.appendNode( 2.5, 2.0 );
-  const axom::IndexType n5 = mesh.appendNode( 5.0, 0.0 );
+  const axom::IndexType n0 = mesh.appendNode(0.0, 0.0);
+  const axom::IndexType n1 = mesh.appendNode(2.0, 0.0);
+  const axom::IndexType n2 = mesh.appendNode(1.0, 1.0);
+  const axom::IndexType n3 = mesh.appendNode(3.5, 1.0);
+  const axom::IndexType n4 = mesh.appendNode(2.5, 2.0);
+  const axom::IndexType n5 = mesh.appendNode(5.0, 0.0);
 
   // Append mesh cells
-  const axom::IndexType c0[ ] = { n0, n1, n2 };
-  const axom::IndexType c1[ ] = { n1, n5, n3, n2 };
-  const axom::IndexType c2[ ] = { n3, n4, n2 };
+  const axom::IndexType c0[] = {n0, n1, n2};
+  const axom::IndexType c1[] = {n1, n5, n3, n2};
+  const axom::IndexType c2[] = {n3, n4, n2};
 
-  mesh.appendCell( c0, mint::TRIANGLE );
-  mesh.appendCell( c1, mint::QUAD );
-  mesh.appendCell( c2, mint::TRIANGLE );
+  mesh.appendCell(c0, mint::TRIANGLE);
+  mesh.appendCell(c1, mint::QUAD);
+  mesh.appendCell(c2, mint::TRIANGLE);
 
-// sphinx_tutorial_construct_unstructured_mixed_end
+  // sphinx_tutorial_construct_unstructured_mixed_end
 
   // STEP 3: write output mesh
-  mint::write_vtk( &mesh, "tutorial_mixed_unstructured_mesh.vtk" );
+  mint::write_vtk(&mesh, "tutorial_mixed_unstructured_mesh.vtk");
 }
 
 //------------------------------------------------------------------------------
 void using_external_storage()
 {
-
-// sphinx_tutorial_using_external_storage_start
+  // sphinx_tutorial_using_external_storage_start
 
   constexpr axom::IndexType NUM_NODES = 6;
   constexpr axom::IndexType NUM_CELLS = 4;
 
   // application buffers
-  double x[ ] = { 0.0, 2.0, 1.0, 3.5, 2.5, 5.0 };
-  double y[ ] = { 0.0, 0.0, 1.0, 1.0, 2.0, 0.0 };
-
-  axom::IndexType cell_connectivity[ ] = {
-    1, 3, 2, // c0
-    2, 0, 1, // c1
-    3, 4, 2, // c2
-    1, 5, 3  // c3
+  double x[] = {0.0, 2.0, 1.0, 3.5, 2.5, 5.0};
+  double y[] = {0.0, 0.0, 1.0, 1.0, 2.0, 0.0};
+
+  axom::IndexType cell_connectivity[] = {
+    1,
+    3,
+    2,  // c0
+    2,
+    0,
+    1,  // c1
+    3,
+    4,
+    2,  // c2
+    1,
+    5,
+    3  // c3
```

Comment:
@gzagaris -- I played around with several configurations for this (as a proxy for several other cases).  
clang-format does not appear to have any automatic options to leave this alone.

It could be a good candidate for `clang-format off` and `clang-format on`, if it bothers you.